donate.html

```diff
@@ -819,7 +902,7 @@ <h4>Shipping Address</h4>
                 total += parseFloat(other);
         }
 
-        var new_total = total == 0 ? "" : "" + total.toFixed(0);
+        var new_total = total == 0 ? "" : "" + (total % 1 != 0 ? total.toFixed(2) : total.toFixed(0));

```

Comment:
strict equal would be better. no?

Reply:
Yeah, strict would be better, that's my bad, I just adapted the second half without changing the initial if.

Reply:
@mrkimani-ndegwa This is fixed in a1ede1c